app/controllers/editorial/editorial_controller.rb

```diff
@@ -9,6 +9,13 @@ class EditorialController < ::ApplicationController
 
     layout 'editorial'
 
+    rescue_from CanCan::AccessDenied do |exception|
```

Comment:
Looks like we'll need to fix this in two other places (application controller and admin controller) too.

I'm also getting concerned about the lack of DRYness in our controllers. We have 3 controller hierarchies :/